<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiVMS2020Class</class>
 <widget class="QMainWindow" name="BiVMS2020Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiVMS2020</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/BiVMS2020.qrc">
    <normaloff>:/test/BiVMS/图像.png</normaloff>:/test/BiVMS/图像.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>项目</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>新建项目</string>
     </property>
     <addaction name="actionRealTime3DProject"/>
     <addaction name="actionAnalysis3DProject"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="actionLoadProject"/>
    <addaction name="actionProjectInformation"/>
    <addaction name="separator"/>
    <addaction name="actionStartPage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>通信</string>
    </property>
    <addaction name="actionCommuincationSetting"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionCalibration"/>
    <addaction name="actionHardwareControl"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionLoadProject">
   <property name="text">
    <string>加载项目</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionProjectInformation">
   <property name="text">
    <string>项目信息</string>
   </property>
  </action>
  <action name="actionRealTime3DProject">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/live.png</normaloff>:/test/BiVMS/live.png</iconset>
   </property>
   <property name="text">
    <string>三维实时</string>
   </property>
  </action>
  <action name="actionAnalysis3DProject">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/分析 (2).png</normaloff>:/test/BiVMS/分析 (2).png</iconset>
   </property>
   <property name="text">
    <string>三维分析</string>
   </property>
  </action>
  <action name="actionCommuincationSetting">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/shujuzhuanxian.png</normaloff>:/test/BiVMS/shujuzhuanxian.png</iconset>
   </property>
   <property name="text">
    <string>设置通信</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/calibration.png</normaloff>:/test/BiVMS/calibration.png</iconset>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </action>
  <action name="actionStartPage">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/welcome.png</normaloff>:/test/BiVMS/welcome.png</iconset>
   </property>
   <property name="text">
    <string>起始页</string>
   </property>
  </action>
  <action name="actionHardwareControl">
   <property name="icon">
    <iconset resource="../resource/BiVMS2020.qrc">
     <normaloff>:/test/BiVMS/辐射.png</normaloff>:/test/BiVMS/辐射.png</iconset>
   </property>
   <property name="text">
    <string>硬件控制</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/BiVMS2020.qrc"/>
 </resources>
 <connections/>
</ui>
